merklize/merklize.go

```diff
@@ -20,13 +20,17 @@ import (
 	"github.com/piprate/json-gold/ld"
 )
 
-var defaultHasher Hasher = PoseidonHasher{}
-
-// ErrorFieldIsEmpty is returned when field path to resolve is empty
-var ErrorFieldIsEmpty = errors.New("fieldPath is empty")
+var (
+	defaultHasher Hasher = PoseidonHasher{}
+	numRE                = regexp.MustCompile(`^\d+$`)
```

Comment:
numRE ?

Reply:
For checking,  if the string is a number.